go.mod

```diff
@@ -23,24 +23,24 @@ require (
 	go.uber.org/zap v1.24.0
 	golang.org/x/sync v0.3.0
 	google.golang.org/grpc v1.56.1
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/client-go v0.25.4
+	k8s.io/api v0.27.2
```

Comment:
This should not be in here, right?

Reply:
It should be no problem. It is ignored as we pin to `k8s.io/api => k8s.io/api v0.25.4` - https://github.com/nak3/serverless-operator/blob/replace-2161/go.mod#L192

It is same with [`knative.dev/eventing v0.37.1`](https://github.com/nak3/serverless-operator/blob/replace-2161/go.mod#L29) on release-v1.10. It should be 0.36 on release-1.10 but just [replaces with the correct version](https://github.com/openshift-knative/serverless-operator/blob/release-1.30/go.mod#L29).